bin/ci-builder

```diff
@@ -67,6 +67,34 @@ else
     rust_components+=,clippy-preview,rustfmt-preview
 fi
 
+# Target a specific CPU to get better performance, i.e. use more modern instructions.
+#
+# All of the x86-64 and ARM hardware we run in production support the v3 and neoverse-n1 micro
+# architectures, respectively.
+X86_64_TARGET_CPU="x86-64-v3"
+AARCH64_TARGET_CPU="neoverse-n1"
```

Comment:
```suggestion
x86_64_target_cpu="x86-64-v3"
aarch64_target_cpu="neoverse-n1"
```

Shell variables should be lower case unless they're exported.

Reply:
Done!